src/main/resources/application.yml

```
@@ -29,4 +34,27 @@ springdoc:
+  # Redis
+  redis:
+    key:
+      prefix: runner                    # 레디스 저장에 필요한 키 값 중 접두어
+      suffix: token_rts                 # 레디스 저장에 필요한 키 값 중 접미어
+    max-token: 1                        # 최대로 발급 가능한 토큰의 수
+
+kakao:
+  client-id: ${KAKAO_CLIENT_ID}
+  redirect-uri: ${KAKAO_REDIRECT_URI}
+  auth-api: https://kauth.kakao.com
+  api: https://kapi.kakao.com
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)